.prettierrc

```diff
@@ -3,7 +3,6 @@
 	"singleQuote": true,
 	"trailingComma": "es5",
 	"printWidth": 100,
-	"pluginSearchDirs": ["."],
```

Comment:
This option has been removed, see: https://prettier.io/blog/2023/07/05/3.0.0.html#plugin-search-feature-has-been-removed-14759httpsgithubcomprettierprettierpull14759-by-fiskerhttpsgithubcomfisker

It works just fine now with the `plugins` config further below.